### PR TITLE
Add LIME on images tutorial (fixes #97)

### LINKED ISSUE #97: Tutorial notebook for LIME for images.
1 example per notebook.
Use the dataset in the exploration repo. 
We want to cover binary MNIST. OR triangles and circles OR LeafSnap datasets and cover all in all notebooks.

### PR DESCRIPTION
**Pull request by loostrum**
The output doesn't look right yet.
Also waiting for a decision on a few other issues that might be nicer to fix before the tutorials (see comments in #97)

### CODE REVIEW

File: tutorials/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Tutorials
+This folder contains tutorial notebooks for DIANNA.
+A tutorial is avaiable for several combinations of data modality and explanability method.
+Click one of the links in the table to directly go to a notebook:
+
+|            | images                     | text |
+| :--        | :--:                       | :--:  |
+| LIME       | [link](lime_images.ipynb)  | []() |
+| RISE       | []()                       | []() |
+| KernelShap | []()                       | []() |
```
Comment (cwmeijer) [inline tutorials/README.md:10]: Elena and I just worked on the Readme. There we have the same table, but transposed. We thought it's more common to have the XAI methods in the columns (although both ways make perfect sense). For consistency, could you do the same for this one?

Comment (loostrum) [reply tutorials/README.md]: Done!

**Comment by loostrum** [comment]
It's more or less done, but I'm struggling with improving the results. Whatever I try, the output seems to remain basically the same.

**Comment by egpbos** [comment]
About the results: perhaps good to look at that later, maybe make an issue for it.

**Comment by loostrum** [comment]
Good idea!

**Review by egpbos — APPROVED** [review]
LGTM! CI failures are not related to this PR, so all green, mergy time.

**Pull request merged**